Header image_header
Detection_result[] detection_results